style.css

```diff
@@ -122,6 +102,81 @@ input, select {
     border: #ffffff solid 2px;
     border-radius: 3px;
 }
+.card-parent {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+}
+.card-parent div {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.card-parent h3 {
+    display: inline-block;
+}
+.card-outer {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 30px;
+}
+.card-inner {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap: 20px;
+}
+.card-inner div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 0;
+}
+.card-inner div p {
+    margin: 0 auto;
+}
+a .anchor {
+    display: block;
+    position: relative;
+    top: -250px;
+    visibility: hidden;
```

Comment:
instead of this magic which is hard to explain, put `scroll-padding-top` on the div and put the id on the div (as discussed)